File: hello-webflux/completable-future/src/main/java/me/kzv/common/entity/UserEntity.java
package me.kzv.common.entity;

public class UserEntity {
    private final String id;
    private final String name;
    private final int age;
    private final String profileImageId;

    public UserEntity(String id, String name, int age, String profileImageId) {
        this.id = id;
        this.name = name;
        this.age = age;
        this.profileImageId = profileImageId;
    }

    public String getId() {
        return id;
    }

    public String getName() {
        return name;
    }

    public int getAge() {
        return age;
    }

    public String getProfileImageId() {
        return profileImageId;
    }
}

File: multi-tenancy/tenancy-manager/src/main/kotlin/me/kzv/tenancymanager/user/entity/UserEntity.kt
package me.kzv.tenancymanager.user.persistence

import jakarta.persistence.Entity
import jakarta.persistence.Table
import me.kzv.common.jpa.BaseEntity

@Table(name = "users")
@Entity
class UserEntity(
    val email: String,
    var name: String,
    var password: String,
) : BaseEntity() {

}